README.md

```diff
@@ -6,7 +6,69 @@
   <a href="https://circleci.com/gh/kids-first/kf-ui-data-tracker"><img src="https://img.shields.io/circleci/project/github/kids-first/kf-ui-data-tracker.svg?style=for-the-badge"></a>
 </p>
 
-Kids First Data Tracker
-=======================
+# Kids First Data Tracker
 
 Investigator study management and progress tracking.
+
+## Development Quick Start
+
+#### Setup backend API service
+
+If you haven't already, create the kf-data-stack network:
+
+```
+docker network create kf-data-stack
+```
+
+Checkout kf-api-study-creator master and bring up the study creator API:
+
+```
+docker-compose up
+```
+
+For details, go to https://github.com/kids-first/kf-api-study-creator
```

Comment:
You could just defer this section to the api repo's README as most of the instructions are there.